code/datums/station_traits/positive_traits.dm

```diff
@@ -270,3 +270,60 @@
 	// Put our filthy fingerprints all over the contents
 	for(var/obj/item/item in wallet)
 		item.add_fingerprint(living_mob, ignoregloves = TRUE)
+
+/datum/station_trait/cybernetic_revolution
+	name = "Cybernetic Revolution"
+	trait_type = STATION_TRAIT_POSITIVE
+	show_in_report = TRUE
+	weight = 1
+	report_message = "The new trends in cybernetics have come to the station! Everyone has some form of cybernetic implant."
+	trait_to_give = STATION_TRAIT_CYBERNETIC_REVOLUTION
+	/// List of all job types with the cybernetics they should receive.
+	var/static/list/job_to_cybernetic = list(
+		/datum/job/assistant = /obj/item/organ/internal/heart/cybernetic, //real cardiac
+		/datum/job/atmospheric_technician = /obj/item/organ/internal/cyberimp/mouth/breathing_tube,
+		/datum/job/bartender = /obj/item/organ/internal/liver/cybernetic/tier3,
+		/datum/job/botanist = /obj/item/organ/internal/cyberimp/chest/nutriment,
+		/datum/job/captain = /obj/item/organ/internal/heart/cybernetic/tier3,
```

Comment:
Maybe an oversight, does granting the cybernetic heart silently remove the respawning ability from Ethereal crewmembers?
I believe that is a function of their alien hearts, which they'd probably prefer to have over this.

Reply:
known, i dont think its a big deal, once in a while your character has gotten a replacement heart and cant revive for a round